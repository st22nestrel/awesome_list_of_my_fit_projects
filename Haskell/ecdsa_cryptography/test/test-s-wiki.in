Curve {
    p: 0x17
    a: 1
    b: 1
    g: Point {
    x: 0x0D
    y: 0x10
    }
    n: 0x7
    h: 1
}
Key {
    d: 0x2
    Q: 0x040513
}
Hash: 5