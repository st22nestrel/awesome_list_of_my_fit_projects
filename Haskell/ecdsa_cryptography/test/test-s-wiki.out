Signature {
	r: 0x0000000000000000000000000000000000000000000000000000000000000005
	s: 0x000000000000000000000000000000000000000000000000000000000000002d
}

